webauthn-authenticator-rs/Cargo.toml

```diff
@@ -58,3 +58,6 @@ tracing-subscriber = { version = "0.3", features = ["env-filter", "std", "fmt"]
 clap = { version = "^3.2", features = ["derive", "env"] }
 tokio = { version = "1.22.0", features = ["sync", "test-util", "macros", "rt-multi-thread", "time"] }
 tempfile = { version = "3.3.0" }
+
+[build-dependencies]
+openssl = "0.10"
```

Comment:
Can this bump to the version we just patched over in the other repo, for consistency/avoiding recompilation/other issues? :D

Reply:
If a dep is in both build and dependencies, it'll merge the version to the parent in dependencies, so the rule of 0.10.41 from deps will apply here so I think leaving this as 0.10 is fine.